master/usage/encrypt-comms.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Configuring encryption and authentication
+canonical_url: 'https://docs.projectcalico.org/master/usage/encrypt-comms'
+---
+
+## Connections from {{site.prodname}} components to etcd
+
+If you are using the etcd datastore, we recommend enabling mutual TLS authentication on
+its connections as follows.
+
+- [Configure etcd](https://coreos.com/etcd/docs/latest/op-guide/security.html) to encrypt its
+  communications with TLS and require clients to present certificates signed by the etcd certificate
+  authority.
+
+- Configure each {{site.prodname}} component to verify the etcd server's identity and to present
+  a certificate to the etcd server that is signed by the etcd certificate authority.
+  - [{{site.nodecontainer}}](../reference/node/configuration)
+  - [`calicoctl`](./calicoctl/configure/etcd)
+  - [CNI plugin](../reference/cni-plugin/configuration#etcd-location) (Kubernetes and OpenShift only)
+  - [Felix](../reference/felix/configuration#etcdv3-datastore-configuration) (on [bare metal hosts](../getting-started/bare-metal/installation/))
+  - [Typha](../reference/typa/configuration#etcdv3-datastore-configuration) (often deployed in
+    larger Kubernetes deployments)
+  - [Neutron or ML2 driver](./openstack/configuration#neutron-server-etcneutronneutronconf) (OpenStack only)
+  - [DHCP agent](https://docs.projectcalico.org/master/getting-started/openstack/) (OpenStack only)
```

Comment:
The config in neutron.conf is good for the DHCP agent too, so please also use the openstack/configuration#neutron-server-etcneutronneutronconf link here.